richtextfx/src/main/java/org/fxmisc/richtext/TextExt.java

```diff
@@ -17,6 +17,11 @@
 import javafx.scene.shape.StrokeLineCap;
 import javafx.scene.text.Text;
 
+/**
+ * A class which adds some more styleable properties to JavaFX's Text class.
+ *
+ * They can be styled using the properties (and accessors/mutators) or via CSS.
+ */
```

Comment:
Perhaps there should be a note here that explains that the added-on CSS properties all start with the "-rtfx" prefix.